WordPressAuthenticator/Sources/Authenticator/WordPressAuthenticatorDisplayStrings.swift

```diff
@@ -176,8 +176,8 @@ public extension WordPressAuthenticatorDisplayStrings {
                                                           comment: "Enter your account information for {site url}. Asks the user to enter a username and password for their self-hosted site."),
             usernamePasswordInstructions: NSLocalizedString("Log in with your WordPress.com username and password.",
                                                             comment: "Instructions on the WordPress.com username / password log in form."),
-            twoFactorInstructions: NSLocalizedString("Please enter the verification code from your authenticator app.",
-                                                     comment: "Instruction text on the two-factor screen."),
+            twoFactorInstructions: NSLocalizedString("Please enter the verification code from your authentication app for your WordPress.com account.",
```

Comment:
```suggestion
            twoFactorInstructions: NSLocalizedString("twoFactorInstructions.details", value: "Please enter the verification code from your authentication app for your WordPress.com account." comment: ""Instruction label in the two-factor authorization screen WordPress.com login authentication. Note: it has to mention that it's for a WordPress.com account."),
```

Reference: https://github.com/wordpress-mobile/WordPress-iOS/blob/trunk/docs/localization.md#use-unique-reverse-dns-naming-style-keys